### ISSUE TITLE
Keyframe configuration for Autoplay Duration and Voiceovers

### ISSUE THREAD

**Issue by whitman** [authorly/interapptive-web#952]
During autoplay, if a voiceover audio file has a duration that is longer than 8 seconds, the audio will get cut off because the number 8 is used as the autoplay duration if there's no text. To resolve this, there could be a configuration icon would appear over a keyframe when hovering over. When the config icon is clicked, it would open up a modal with a form (similar to app settings). 

The hardcoded `8` @ https://github.com/curiousminds/interapptive-web/blob/master/app/assets/javascripts/app/models/keyframe.js.coffee#L113 should be a value retrieved from the keyframe model as one of its fields. If a user does change the value in this form, that value should take precedence over the values that are obtained from the duration or word count. 

Also, the voiceover duration should be used as the `autoplayDuration` node value, even if there are no texts. Currently, if there are 0 texts, it will use the hardcoded value `8` (which will be changed to a field that the user can edit)

Correlates with https://github.com/curiousminds/interapptive/issues/277
